backend/wargamebackend/wargamelogic/views/get.py

```
@@ -202,16 +230,17 @@ def get_game_unit_instances_by_team_name_and_branch(request, join_code, team_nam
     return Response(serializer.data)
 
 @api_view(['GET'])
+@authentication_classes([CookieJWTAuthentication])
 @permission_classes([IsAuthenticated])
-def get_game_tiles_for_landmark_instance_by_id(request, join_code, pk):
-    game_instance = get_object_or_404(GameInstance, join_code=join_code)
-    landmark_instance = get_object_or_404(LandmarkInstance, pk=pk, game_instance=game_instance)
+def get_game_tiles_for_landmark_instance_by_id(request, pk):
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)